File: sandbox/ruby/src/AccountVerifyDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

account_verify_request = Dropbox::Sign::AccountVerifyRequest.new
account_verify_request.email_address = "some_user@dropboxsign.com"

begin
    response = Dropbox::Sign::AccountApi.new.account_verify(
        account_verify_request,
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling Account#account_verify: #{e}"
end

File: sandbox/php/src/AccountVerifyDefaultExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();

$account_verify_request = (new Dropbox\Sign\Model\AccountVerifyRequest())
    ->setEmailAddress("some_user@dropboxsign.com");

try {
    $response = (new Dropbox\Sign\Api\AccountApi(config: $config))->accountVerify(
        account_verify_request: $account_verify_request,
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling Account#accountVerify: {$e->getMessage()}";
}
